docs/patches/upstream-status.md

```
@@ -6,21 +6,38 @@ Progress toward getting these fixes into the mainline Linux kernel.
 
 ## Submission History
 
-### v7 (Current)
+### v7 (Current - January 29, 2026)
 
-- **Date:** January 2026
-- **Series:** 12 patches
-- **Status:** Under review on linux-wireless mailing list
+- **Date:** January 29, 2026
+- **Series:** 6 patches
+- **To:** Felix Fietkau (nbd@nbd.name) - MT76 maintainer
```

markdownlint-cli2 (0.20.0)

Bare URL used

(MD034, no-bare-urls)